site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -544,3 +543,36 @@
 </script>
 
 
+<script>
+    window.onload = function() {
```

Comment:
You may want to move this logic into this file https://github.com/Submitty/Submitty/blob/main/site/ts/grader-submission.ts#L23 which handles a lot of UI controls for bulk upload already, and some of the logic added here might be duplicated there / attempted to change the style as well

Reply:
yes, thanks for your advice, initially I was editing mentioned ts file, but then I got lazy with some trivial syntax, which I admit is my fault.